pkg/nexus/nexus_test.go

```diff
@@ -147,3 +78,70 @@ func createZipFile() (bytes.Buffer, error) {
 	}
 	return *buf, nil
 }
+
+func TestMavenDownloaderOnSnapshot(t *testing.T) {
```

Comment:
Jeg savner en test for nedlasting av releases også. (c.Snapshot==false)

Reply:
Den finner du i TestMavenDownloaderOnRelease

Reply:
hm. Jeg skjønner nå at på releases skal vi ikke hente noe fra maven. Det er vel det som klages over i #aurora-kanalen fra gld.